### DISCUSSION TITLE
Using CSLA Version 8 Overview book online

### DISCUSSION THREAD

**Discussion by rockfordlhotka** [MarimerLLC/csla#3615] [Announcements]
I have released the free _Using CSLA Version 8 Overview_ ebook online.

https://store.lhotka.net/using-csla-version-8-overview

This took a lot longer than anticipated, and I want to thank @TheCakeMonster and @royog for being reviewers, and for their patience as I ever so slowly worked my way through the content.

<img width="611" alt="UsingCsla-Overview_cover" src="https://github.com/MarimerLLC/csla/assets/2333134/20611545-bae1-40a6-8959-f86f5cf3b8d4">

**Comment by AbbasMalik** [comment]
When is "Using CSLA Version 9" expected?

**Reply by rockfordlhotka** [reply]
I don't have a planned timeline yet. Probably in the next 2-3 months.
